lib/matplotlib/tests/test_image.py

```diff
@@ -1462,6 +1459,45 @@ def test_rgba_antialias():
                   cmap=cmap, vmin=-1.2, vmax=1.2)
 
 
+@check_figures_equal(extensions=('png', ))
+def test_upsample_interpolation_stage(fig_test, fig_ref):
+    """
+    Show that interpolation_stage='antialiased' gives the same as 'data'
+    for upsampling.
+    """
+    # Fixing random state for reproducibility.  This non-standard seed
+    # gives red splotches for 'rgba'.
+    np.random.seed(19680801+9)
+
+    grid = np.random.rand(4, 4)
+    ax = fig_ref.subplots()
+    ax.imshow(grid, interpolation='bilinear', cmap='viridis',
+              interpolation_stage='data')
+
+    ax = fig_test.subplots()
+    ax.imshow(grid, interpolation='bilinear', cmap='viridis',
+              interpolation_stage='antialiased')
+
+
+@check_figures_equal(extensions=('png', ))
+def test_downsample_interpolation_stage(fig_test, fig_ref):
+    """
+    Show that interpolation_stage='antialiased' gives the same as 'rgba'
+    for downsampling.
+    """
+    # Fixing random state for reproducibility
+    np.random.seed(19680801)
+
+    grid = np.random.rand(4000, 4000)
+    ax = fig_ref.subplots()
+    ax.imshow(grid, interpolation='antialiased', cmap='viridis',
```

Comment:
```suggestion
    ax.imshow(grid, interpolation='hanning', cmap='viridis',
```

leave nothing to chance?

Reply:
Well the opposite - I'd like this to fail if we change the default, so at least we do it purposefully?

Reply:
fair.